### ISSUE TITLE
How to set values for `parametrize` programmatically / using `$ext` ?

### ISSUE THREAD

**Issue by dhpnt** [taverntesting/tavern#781]
Hi,
Is there a way I can define values for `parametrize` programmatically?

- Instead of,
```
marks:
  - parametrize:
      key:
        - foo
        - bar
      vals:
        - [a, 1]
	- [a, 2]
	- [z, 3]
	- [x, 99]
```

- Something like,
```
marks:
  - parametrize:
      key:
        - foo
        - bar
      vals:
        $ext:
          function: utils:get_parametrize_vals
```
I am getting `tavern.util.exceptions.BadSchemaError: 'vals' should be a list error` if I try using $ext at this level.
TIA!

**Comment by dhpnt** [comment]
@michaelboulton , Could you please help me out with this issue?

**Comment by michaelboulton** [comment]
This can't be done at the moment, I don't particularly want to change this before the 2.0 version because it will mean messing around more with the schema verification which will mean duplication of work.

**Comment by michaelboulton** [comment]
This is fixed in 2.0.0alpha14 (and won't be backported to 1.0)
